study-builder/studies/testboston/snippets/question-sample-collect-time-min.conf

```
@@ -0,0 +1,30 @@
+{
+  include required( "../../snippets/numeric-question.conf"),
+  "stableId": ${id.q.sample_collect_time_min},
+  "hideNumber": true,
+  "promptTemplate": {
+    "templateType": "HTML",
+    "templateText": "$sample_collect_time_min_prompt",
+    "variables": [
+      {
+        "name": "sample_collect_time_min_prompt",
+        "translations": [
+          { "language": "en", "text": "Minutes" },
```

Per Jen's request in Slack, here should be:
```suggestion
          { "language": "en", "text": "Minute" },
```